tutorials/transformations_02_coordinate_system_manager.ipynb

```diff
@@ -9,18 +9,6 @@
     "## Imports"
```

Comment:
<p>Nowadays, we call it Jupyter notebook :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BAMWelDX/weldx/pull/767/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>;) changed it</p>